### PR TITLE
Wait for both cmc controllers to update

### PR DESCRIPTION
**Pull request by logankimmel**

### CODE REVIEW

File: lib/puppet/provider/cmc_fw_update/racadm.rb [hunk 1]
```diff
@@ -109,9 +110,16 @@ def create
...
+    update_cycles = @partitions.size
+    update_cycles.times do
+      until status == "ready"
+        status = update_status?
+        sleep 40
+      end
+      if update_cycles > 1
+        Puppet.debug("Multiple controllers found.  Sleeping and waiting for second to update")
+        sleep 180
+      end
```
Comment (gavin-scott) [inline lib/puppet/provider/cmc_fw_update/racadm.rb]: I think it may be safer to just run each of the controller updates separately.

**Comment by logankimmel** [comment]
@gavin-scott updated

File: lib/puppet/provider/cmc_fw_update/racadm.rb [hunk 2]
```diff
@@ -24,39 +24,51 @@ def get_current_version(fw_version)
...
     output.each_line do |l|
-      if l.start_with? 'Primary CMC Version'
-        versions[:primary] = l.split('=')[1].gsub(' ','').chop
-        @partitions << '-m cmc-active'
+      if l.start_with? 'Primary CMC Location'
+        location = l.split('=')[1].strip
+        primary[:name] = location
+        location == "CMC-1" ? standby[:name] = "CMC-2" : standby[:name] = "CMC-1"
+      elsif l.start_with? 'Primary CMC Version'
+        primary[:version] = l.split('=')[1].gsub(' ','').chop
```
Comment (gavin-scott) [inline lib/puppet/provider/cmc_fw_update/racadm.rb]: @logankimmel says this can come out

Comment (logankimmel) [inline lib/puppet/provider/cmc_fw_update/racadm.rb]: @gavin-scott Let's keep this in here.  I'm using it on line 118

**Pull request merged**